samples/HelloOpenGL/HelloOpenGL.vsimporter/HelloOpenGL-WinStore10/HelloOpenGL.vcxproj

```diff
@@ -29,74 +28,78 @@
     <EnableDotNetNativeCompatibleProfile>true</EnableDotNetNativeCompatibleProfile>
     <WindowsTargetPlatformVersion>10.0.10586.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.10586.0</WindowsTargetPlatformMinVersion>
-    <ProjectGuid>{94BA35CD-F4E9-4B5B-BEE7-A452AB514360}</ProjectGuid>
-    <WINOBJC_SDK_ROOT>..\..\..\..</WINOBJC_SDK_ROOT>
+    <Keyword>IslandwoodProj</Keyword>
+    <ProjectGuid>{5F8B73C6-1253-4481-9567-A81321309725}</ProjectGuid>
+    <RootNamespace>HelloOpenGL</RootNamespace>
+    <WINOBJC_SDK_ROOT Condition="'$(WINOBJC_SDK_ROOT)' == ''">E:\island\github\henryfox\WinObjC</WINOBJC_SDK_ROOT>
```

Comment:
>E:\island\github\henryfox\WinObjC [](start = 62, length = 33)

Please be sure to import using -relativepath, or replace with ..\..\..\...  Here and elsewhere.